importlab/fs.py

```diff
@@ -1,13 +1,17 @@
 import abc
+import glob
 import os
 import tarfile
+import tempfile
+
+from six import with_metaclass
```

Comment:
no need for this, we dropped py2 compatibility in pytype a while ago

Reply:
Sorry about that!